public/static/docs/command-reference/commit.md

```diff
@@ -21,7 +21,10 @@ The `dvc commit` command is useful for several scenarios where a dataset is
 being changed: when a [stage](/doc/command-reference/run) or
 [pipeline](/doc/command-reference/pipeline) is in development, when one wishes
 to run commands outside the control of DVC, or to force DVC-file updates to save
-time tying stages or a pipeline.
+time tying stages or a pipeline. These scenarios are further detailed below.
+
+💡 For convenience, a Git hook is available to remind you to `dvc commit` when
```

Comment:
I think it does not remind to this, it's just a regular status message.

Reply:
Will follow up on all these notes (it's in several cmd refs) in https://github.com/iterative/dvc.org/pull/931#pullrequestreview-351859026 above.